packages/api/index.ts

```diff
@@ -30,5 +31,11 @@ type RouterInputs = inferRouterInputs<AppRouter>;
  **/
 
 type RouterOutputs = inferRouterOutputs<AppRouter>;
-export { createTRPCContext, appRouter, createCaller, checkRateLimitThrowError };
+export {
+  createTRPCContext,
+  appRouter,
+  createCaller,
+  checkRateLimitThrowError,
+  sendMail,
```

Comment:
Do not export this function here? I already told you right now we are not using it anywhere.